test/e2e/specs/liquidation.spec.js

```diff
@@ -124,5 +129,63 @@ describe('Wallet App Test Cases', () => {
         /Please increase your collateral or repay your outstanding IST debt./,
       );
     });
+
+    it('should wait and verify vaults are being liquidated', () => {
+      cy.contains(/3 vaults are liquidating./, {
+        timeout: LIQUIDATING_TIMEOUT,
+      });
+    });
+
+    it('should verify the value of startPrice from the CLI successfully', () => {
+      const fieldName = 'book0.startPrice';
+      const expectedValue = '9.99 IST/ATOM';
+
+      cy.checkFieldValue(fieldName, expectedValue).then(isMatched => {
+        expect(isMatched).to.be.true;
+      });
```

Comment:
could the callback be an async function?
```suggestion
      expect(await cy.checkFieldValue(fieldName, expectedValue)).to.be.true;
```

or could `checkFieldValue` be an `assertFieldValue` and throw when they don't match?

Reply:
Cypress tends to have trouble with async awaits. Please see [this](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#The-Cypress-Command-Queue).